package-lock.json

```diff
@@ -13,11 +13,11 @@
         "dotenv": "^16.4.7",
         "hono": "^4.0.9",
         "openai": "^4.79.3",
-        "spinai": "*",
-        "tsx": "^4.7.1"
+        "spinai": "*"
```

Comment:
spinai: '*'? nahh that's sus 🤔 pin your deps fam before they upd8 and flop on u fr.